cloud/test/meta_service_http_test.cpp

```diff
@@ -1625,4 +1629,69 @@ TEST(MetaServiceHttpTest, QueryRateLimit) {
     }
 }
 
+TEST(MetaServiceHttpTest, UpdateConfig) {
+    HttpContext ctx;
+    {
+        auto [status_code, content] = ctx.query<std::string>("update_config", "");
+        ASSERT_EQ(status_code, 400);
+        std::string msg = "query param `config` should not be empty";
+        ASSERT_NE(content.find(msg), std::string::npos);
+    }
+    {
+        auto [status_code, content] = ctx.query<std::string>("update_config", "configs=aaa");
+        ASSERT_EQ(status_code, 400);
+        std::string msg = "config aaa is invalid";
+        ASSERT_NE(content.find(msg), std::string::npos);
+    }
+    {
+        auto [status_code, content] = ctx.query<std::string>("update_config", "configs=aaa=bbb");
+        ASSERT_EQ(status_code, 400);
+        std::string msg = "set aaa=bbb failed";
+        ASSERT_NE(content.find(msg), std::string::npos);
+    }
+    {
+        auto [status_code, content] =
+                ctx.query<std::string>("update_config", "configs=recycle_interval_seconds=3599");
+        ASSERT_EQ(status_code, 200);
+        ASSERT_EQ(config::recycle_interval_seconds, 3599);
```

Comment:
also check some config with other types string and bool
e.g. 
s3_client_http_scheme, "http"
enable_cluster_name_check, "false"